app/models/journeyDomain/item/additionalInformation/AdditionalInformationDomain.scala

```diff
@@ -31,12 +31,14 @@ case class AdditionalInformationDomain(
 )(itemIndex: Index, additionalInformationIndex: Index)
     extends JourneyDomainModel {
 
+  def asString: String = s"${`type`} - $value"
```

Comment:
Let's refrain from doing asString and override the toString method instead. Think Bledar might have already done this in his branch actually